# Mule-Deer-CO-ddRAD

# Yet again, full of hope

# And once more

In general, I am interested in population genomic analyses, much like the leopard paper by Pecnerova et al (2021).  Most recently, I worked on a large-scale field project on the effects of chronic wasting disease (CWD) on mule deer populations in northern Colorado, what we called the Laramie Hills Mule Deer Project (URL still exists:  https://www2.nrel.colostate.edu/projects/modelingCWD/index.html).  My part of the project was to carry out population genetic analyses of mule deer populations in the study are in north-central Colorado.

As background, CWD is one of the prion diseases, an inevitably fatal neurodegenerative condition caused by a constitutively expressed protein that misfolds, and called the transmissible spongiform encephalopathies (TSEs).  The same protein is found in all vertebrates, and its misfolding is responsible for TSEs in many mammalian species, including humans (Creutzfeldt-Jacob disease), sheep (scrapie), cattle (mad-cow disease), mink TSEs, and CWD in many cervid species including deer, elk, and moose.  The causative agent, the prion precursor protein (PrPn), is a relatively small (~225 amino acids) cell surface protein that is thought to transport cations like CU+ internally into cells, but especially in lymphatic and nervous tissues.  When the protein misfolds, it loses its native function and becomes non-degradable in proteins plaques within cells.  The buildup of plaques eventually becomes toxic within cells, leading to the loss of nerve cells and neurodegeneration (i.e. spongiform encephalopathy).  

Prion diseases can arise in several ways.  First, PrPn can naturally and spontaneously misfold.  In humans, called Creutzfeldt-Jacob disease (CJD), the rate is approximately 1 in 5 million, and CJD is one of the causes of old-age dementia.  But the most pernicious part of prion diseases is that the misfolded proteins are transmissible as prions are trafficked from cells and become infectious among individuals, spreading the disease within populations.  Often this is by ingesting meat and tissues of a prion-positive individuals (e.g. mad-cow disease in the UK).  In the case of CWD-infected deer, transmission is via saliva, urine, or feces.  The current outbreak of CWD in North America on mule deer, white-tailed deer, elk, and moose initially began in Fort Collins in the 1960’s and has broadly spread among these served species throughout the U.S, and Canada.  Its true origin, whether spontaneous or transmitted from another species like sheep, is unknown (see Figure 1).

   

![image](https://github.com/mfantolin7/Mule-Deer-CO-ddRAD/assets/157053197/fd7d74a3-507d-445d-9222-5ae490f13c4d)


Figure 1:  USGS map of the distribution of CWD in North America, as of March, 2024.  The red oval shows the sampling locations of deer captured as part of the Laramie Foothills Mule Deep Project (2010-2014).  

Another universal feature of prion proteins are the polymorphisms that exist within PrPn, and the underlying gene Prp.  These arise mostly as single-nucleotide mutations that result in single amino acid changes, and where in Prp the mutations occur is different in different species.  But the alternative alleles in most species are known to alter the course of disease, with slower or faster progression of symptoms and times to death associated with different alleles. 

We carried out this study to understand potential transmission pathways, and to assess the effects of alternative Prp alleles on mule deer survival after infection.  The study spanned five years (2009-2013) and includes ~200 female deer sampled (captured, radio-collared and recaptured) and evaluated for CWD infection.  These deer were genotyped for 14 microsatellite loci (unpublished) and revealed what we call “cryptic” population structure, where different areas of the study are genetically similar, but that we can identify genetic matrilines within each part of the study area.    

This genotyping has been followed up with ddRAD sequencing, that revealed ~ 17,000 SNPs.  Population genetic analyses using the RAD-tags have not been completed.  

Since the time of building and processing the ddRAD libraries, two reference genomes for mule deer have been created.  The goal of this project is to create a pipeline for re-analysis of the ddRAD data with respect to these reference genomes.  I have access to the original raw sequences from the ddRAD libraries for mapping of variant RAD-tags to the reference genomes.

Although the full population genomic analysis will not be possible in the time frame or scope of this class, other data included: age, precise location data, CWD status and whether deer became infected during the five-year time course of this study.  We have also and Sanger-sequenced genotypes of the PrP the gene.


